dojo/templates/dojo/metrics_panel.html

```diff
@@ -0,0 +1,12 @@
+
+{% extends "base.html" %}
+{% load event_tags %}
+{% load i18n %}
+{% load display_tags %}
+{% block content %}
+    {{ block.super }}
+    <div class="embed-responsive embed-responsive-16by9"> 
+        <iframe class="embed-responsive-item" src="{{ grafana_url }}/d/edig6ul0oqzuod/panel-de-control-vultracker?orgId=1&from=now-30d&to=now&var-limit=1000&var-offset=1&kiosk" frameborder="0" allowfullscreen></iframe>
```

Comment:
it posible map as a variable "grafana_url }}/d/edig6ul0oqzuod/panel-de-control-vultracker?orgId=1&from=now-30d&to=now&var-limit=1000&var-offset=1&kiosk" ?

Reply:
It would need to be mapped to 3 different variables.
grafana_url = http://localhost:3000
grafana_slug = /d/edig6ul0oqzuod/panel-de-control-vultracker
grafana_params= ?orgId=1&from=now-30d&to=now&var-limit=1000&var-offset=1&kiosk